.travis.yml

```diff
@@ -14,9 +14,20 @@ env:
 
 before_install:
   # Docker-compose installation
-  - curl -L https://github.com/docker/compose/releases/download/1.8.1/docker-compose-`uname -s`-`uname -m` > docker-compose
-  - chmod +x docker-compose
-  - sudo mv docker-compose /usr/local/bin
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
+  - docker-compose --version
+  - docker --version
+
+before_script:
+#  - composer self-update
+#  - composer install
+##  - docker-compose build
+  - sudo sysctl -w vm.max_map_count=262144
+  - docker-compose up -d
```

Comment:
yep!  the docker-compose up -d has been removed from travis now is in the make file